Makefile

```
@@ -184,17 +187,22 @@ eclint/lint:
 	$(ECLINT_FILES) | grep -zv ".bats" | xargs -0 -I {} eclint check {}
 	@ echo "[$@]: Project PASSED eclint test!"
 
-python/%: FIND_PYTHON := find . $(FIND_EXCLUDES) -name '*.py' -type f
-## Lints Python files
-python/lint: | guard/program/black
+python/%: PYTHON_FILES := git ls-files '*.py'
+## Checks format and lints Python files.  Runs pylint on each individual
+## file and uses a custom format for the lint messages.
+python/lint: | guard/program/pylint guard/program/black guard/program/git
+python/lint:
 	@ echo "[$@]: Linting Python files..."
-	$(FIND_PYTHON) | $(XARGS) black --check $$(dirname {})
+	$(PYTHON_FILES) | xargs black --check
+	$(PYTHON_FILES) | ${XARGS} -n1 pylint -rn -sn \
```

prefer to consistently use the make syntax for variables, instead of shell syntax

```suggestion
	$(PYTHON_FILES) | $(XARGS) -n1 pylint -rn -sn \
```

---

Ok. Will do.